src/test/java/healthy/lifestyle/backend/workout/controller/ExerciseControllerTest.java

```diff
@@ -391,4 +391,53 @@ void getDefaultExercisesTest_shouldReturnListOfExercisesAnd200Ok() throws Except
                     .isEqualTo(responseDto.get(id).getHttpRefs());
         });
     }
+
+    @Test
+    void getDefaultExerciseTest_ShouldReturnExerciseAnd200Ok() throws Exception {
+        // Given
+        int seed = 1;
+
+        List<BodyPart> bodyParts = IntStream.rangeClosed(1, 1)
+                .mapToObj(id -> dataHelper.createBodyPart(id))
+                .toList();
+
+        List<HttpRef> httpRefs = IntStream.rangeClosed(1, 1)
+                .mapToObj(id -> dataHelper.createHttpRef(id, false))
+                .toList();
+
+        Exercise defaultExercise =
+                dataHelper.createExercise(seed, false, false, new HashSet<>(bodyParts), new HashSet<>(httpRefs));
+        long id = defaultExercise.getId();
```

Comment:
Add one more default exercise to imitate a real-life scenario, so you can check whether your repository's sql works fine when there are multiple exercises in a database.